apps/kol-amelamdim/pages/admin/dashboard/add-weekly-article/index.tsx

```diff
@@ -39,11 +41,75 @@ export const StyledGrid = styled(Grid)`
   }
 `;
 
+const getWeeklyArticleById = async (id: string) => {
+  try {
+    const article = await axios.get(`/api/admin/get-article-by-id?id=${id}`);
+    if (article.data) {
+      return article.data;
+    }
+  } catch (e) {
+    return null;
```

Comment:
We dont want to do something with that error ?

Reply:
No beacuse if no id found there is going to be a regular "add" instead of edit. that's fine